Dockerfile

```diff
@@ -164,20 +164,15 @@ USER root
 
 ### GO
 
-# Install Go and dep
+# Install Go
 ARG GOLANG_VERSION=1.16.2
 ARG GOLANG_CHECKSUM=542e936b19542e62679766194364f45141fde55169db2d8d01046555ca9eb4b8
-ENV PATH=/opt/go/bin:$PATH \
-  GOPATH=/opt/go/gopath
```

Comment:
This one I wasn't entirely sure of... there were other references to `gopath` folder, but from a quick skim they looked unrelated to this particular folder... it's tricky because `gopath` is also a convenient name to use for a generic folder of where `go` stuff is.

At a higher level the `GOPATH` concept is going away completely in the upcoming `go 1.17`: https://blog.golang.org/go116-module-changes so it'd be best to make it so dependabot works w/o needing `GOPATH` (as traditionally used by `go`) to be set.

I think we should just see what CI says, and if CI greenlights this, then we are probably good to go (assuming CI uses this Dockerfile).